src/main/java/com/checkmarx/flow/service/ConfigurationOverrider.java

```diff
@@ -402,6 +402,10 @@ private static boolean canOverrideBugTracker(BugTracker bugTrackerFromScanReques
         } else if (bugTrackerNameOverride.equalsIgnoreCase(currentBugTrackerType.toString())) {
             cannotOverrideReason = "bug tracker type in override is the same as in scan request";
         }
+        else if (bugTrackerNameOverride.equalsIgnoreCase(BugTracker.Type.NONE.toString())) {
```

Comment:
Shouldn't we handle the BugTracker.Type.NONE case inside `if (comingFromPullRequest)`?

Also, the current code will lead to a strange log message: "Bug tracker override was not applied, because overriding bug tracker to 'None'."

Reply:
Thanks @alex-ko-dev 
The event indeed coming from pull request. so we shouldn't change the the result of the method in case of different bug-tracker

actually the logic is wrong. if bug-tracker=none, the reason should be 'null'
I will fix it